rules/same-core-dependency-version/same-core-dependency-version.js

```diff
@@ -38,18 +41,15 @@ module.exports = {
     if (!packageJson.dependencies) {
       return {};
     }
-    const coreDependencies = getCoreDependencies(process.env.GITHUB_TOKEN);
 
-    for (const [dependency, version] of Object.entries(packageJson.dependencies)) {
-      const ignoreOption = context.options[0];
-      if (
-        ignoreOption && ignoreOption.ignore.length > 0 &&
-        ignoreOption.ignore.includes(dependency)
-      ) {
-          continue;
-      }
+    const options = context.options[0] || {};
```

Comment:
A question and a suggestion.
Where is the `create` function called from? Can we expect the `context` to not have the `options` array at all? In that case this should be
```suggestion
    const options = (context.options && context.options[0] )|| {};
```

Reply:
`create` is called by the ESLint API, and `context.options` will always be an array.